.github/workflows/ci.yaml

```diff
@@ -80,3 +80,47 @@ jobs:
       with:
         name: e2e-sharded
         path: /tmp/e2e/**/artifacts/
+
+  e2e-tests-ppc64le:
+    name: e2e-tests-ppc64le
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: uraimo/run-on-arch-action@v2.2.0
+        name: E2E KCP tests
+        id: build-ppc
+        with:
+          arch: ppc64le
+          distro: ubuntu20.04
+          
+          dockerRunArgs: |
+            --volume "/var/run/docker.sock:/var/run/docker.sock"
+          
+          run: |
+            apt-get update -y
+            apt-get install wget tar gcc git curl runc iptables libdevmapper1.02.1 make jq -y
+            
+            #Install docker
+            mkdir /root/docker; cd /root/docker
+            wget https://oplab9.parqtec.unicamp.br/pub/ppc64el/docker/version-20.10.0/ubuntu-bionic/docker-ce_20.10.0~3-0~ubuntu-bionic_ppc64el.deb
+            wget https://oplab9.parqtec.unicamp.br/pub/ppc64el/docker/version-20.10.0/ubuntu-bionic/docker-ce-cli_20.10.0~3-0~ubuntu-bionic_ppc64el.deb
+            wget https://oplab9.parqtec.unicamp.br/pub/ppc64el/docker/version-20.10.0/ubuntu-bionic/docker-ce-rootless-extras_20.10.0~3-0~ubuntu-bionic_ppc64el.deb
+            wget https://oplab9.parqtec.unicamp.br/pub/repository/debian/ppc64el/containerd/containerd-1.6.4-ppc64le.deb
+            dpkg -i docker-ce_20.10.0~3-0~ubuntu-bionic_ppc64el.deb docker-ce-cli_20.10.0~3-0~ubuntu-bionic_ppc64el.deb docker-ce-rootless-extras_20.10.0~3-0~ubuntu-bionic_ppc64el.deb containerd-1.6.4-ppc64le.deb
+            
+            #Install Go, Kind, kubectl
+            cd /root/ && wget https://go.dev/dl/go1.18.linux-ppc64le.tar.gz
+            tar -C /usr/local -xzf go1.18.linux-ppc64le.tar.gz
```

Comment:
There is a verify script to check go versions used in the project. Please add this one into that script.

Reply:
@sttts can you point me to the script?

Reply:
hack/verify-go-versions.sh